syncapi/sync/request.go

```diff
@@ -32,13 +31,19 @@ import (
 const defaultSyncTimeout = time.Duration(0)
 const defaultTimelineLimit = 20
 
+var (
+	// ErrNotStreamToken is returned if a pagination token isn't of type
+	// types.PaginationTokenTypeStream
+	ErrNotStreamToken = fmt.Errorf("The provided pagination token has the wrong prefix (should be s)")
```

Comment:
I might have missed it, but where do we return this error? Also, why wouldn't a translation from topological token to a stream token be possible if the token's type isn't the one we expect?

Reply:
This looks like a hangover from `SyncPosition` - there's `types.ErrInvalidPaginationTokenType` which is used instead. I've removed it.